transport/interface.go

```diff
@@ -0,0 +1,16 @@
+package transport
+
+import (
+	"context"
+
+	"github.com/filecoin-project/boost/transport/types"
+)
+
+type Transport interface {
+	Execute(ctx context.Context, transportInfo []byte, dealInfo *types.TransportDealInfo) (th Handler, err error)
+}
+
+type Handler interface {
+	Sub() chan types.TransportEvent
+	Close() error
```

Comment:
It looks from the code like we ignore the error return value of `Close() error` so I think we should just remove it

Reply:
Hey, we already discussed this in https://github.com/filecoin-project/boost/pull/40#discussion_r750179349

Reply:
Yeah I know, but we're not actually using the return value in the calling code, AFAICT. So if we're not using the return value, and we're not returning anything from the existing transport implementation, I think we should just remove it.
If in future we need it for a different transport implementation, we can add it in again.

If you feel strongly about it I'm fine keeping it here.

Reply:
I'm okay either ways if you feel so strongly about this. Removed.